partner_phone_extension/views/res_partner_view.xml

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Not blocking, but xmlid's of views should be the same as the inherited views (they are where needed qualified by the module name anyway). The name can be used to indicate the purpose of the extention.

Reply:
I don't understand, Can you more explain this.

Sorry for delay replay.

Reply:
There is not much to explain. All the views should have the same xmlid in id= and in inherit_id=, except that inherit_id= will also have the name of the orginal module in it, and the xmlid in id= should be unqualified. The ir.ui.view name field can be used to describe what the extention or inheriting view's purpose is.